index.bs

```diff
@@ -325,42 +320,62 @@ spec:css-syntax-3;
         <small>(in alphabetical order)</small></th>
         <th><dfn for="credential type registry">Options Member Identifier</dfn></th>
         <th><dfn for="credential type registry">Appropriate Interface Object</dfn></th>
+        <th><dfn for="credential type registry">Get Permissions Policy</dfn></th>
+        <th><dfn for="credential type registry">Create Permissions Policy</dfn></th>
         <th>Specification</th>
         <th>Requestor Contact</th>
       </tr>
     </thead>
+    <tr>
+        <td>digital-credential</td>
+        <td>digital</td>
+        <td>{{DigitalCredential}}</td>
+        <td>digital-credentials-get</td>
+        <td>null</td>
+        <td>[[DIGITAL-CREDENTIALS]]</td>
+        <td><a href="https://wicg.io/">WICG</a></td>
+    </tr>
     <tr>
         <td>federated</td>
         <td>federated</td>
         <td>{{FederatedCredential}}</td>
+        <td>null</td>
+        <td>null</td>
         <td>This specification: [[#federated]]</td>
         <td><a href="https://www.w3.org/2011/webappsec/">W3C</a></td>
     </tr>
     <tr>
         <td>identity</td>
         <td>identity</td>
         <td>{{IdentityCredential}}</td>
+        <td>[=identity-credentials-get=]</td>
+        <td>null</td>
         <td>[[FEDCM]]</td>
         <td><a href="https://www.w3.org/community/fed-id/">W3C</a></td>
     </tr>
     <tr>
         <td>otp</td>
         <td>otp</td>
         <td>{{OTPCredential}}</td>
+        <td>otp-credentials</td>
```

Comment:
should we link to https://wicg.github.io/web-otp/#otp-credentials-feature?

Reply:
Yes, I need to get it exported. Hi @samgoto!

Reply:
Actually, it is exported... but it's not in the database xref for some reason.... looking into it.

Reply:
@dontcallmedom or @tidoust, any idea why https://wicg.github.io/web-otp/#otp-credentials-feature is not ending up in Reffy (or the other service)?

Reply:
The `<dfn>` has a `data-lt` attribute, which overrides the text content. It gets recorded accordingly as [`otp-credentials-feature`](https://respec.org/xref/?term=otp-credentials-feature).

Reply:
```suggestion
        <td>[=otp-credentials|otp-credentials-feature=]</td>
```